core/src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -61,7 +61,6 @@ default void perform(InputSocket<?>[] inputs, OutputSocket<?>[] outputs, Optiona
         perform(inputs, outputs);
     }
 
-    @Deprecated
```

Comment:
1. Most operations don't use the data parameter, it was just a quick hack to let us cache some Mats.
2. Because I need to use this method in this PR